Interfaces/ILoggerService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LibraryApp.Interfaces
+{
+    internal interface ILoggerService
+    {
+        void LogInfo(string message);
+        void NotifyBookAdded();
+        void NotifyBookExists();
+        void NotifyBookDeleted();
```

Comment:
All this methods are doing the same. Its enough to have one `void Log(string message)`